### PR TITLE
Fix readme and Renamed DataCenter to cloud and fixed method names to …

### PR DESCRIPTION
**Pull request by MattB-msft**
Updated Readme,  Updated property and methods names to be consistent with cloud terminology.

### CODE REVIEW

**Review by JimDaly — APPROVED** [review]

File: cds/orgsvc/C#/DiscoveryService/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ You must first set the `username` and `password` variables in the `SampleProgram
 
 ## What this sample does
 
-This sample uses the SDK Assembly `DiscoveryServiceProxy` to query the discovery service with a user's credentials to determine which environments they can connect with.
+This sample uses the SDK Assembly `CrmServiceClient` to query the global discovery service with a user's credentials to determine which environments they can connect with.
```
Comment (phecke) [inline cds/orgsvc/C#/DiscoveryService/README.md:15]: Should be "This sample uses a `CrmServiceClient` object...". CrmServiceClient is not an assembly.

**Review by phecke — CHANGES_REQUESTED** [review]
Minor text change in the README. Otherwise, looks good.

File: cds/orgsvc/C#/DiscoveryService/README.md (hunk 1 above)
Comment (phecke) [reply cds/orgsvc/C#/DiscoveryService/README.md:15]: You could also make 'CrmServiceClient' a link to https://docs.microsoft.com/dotnet/api/microsoft.xrm.tooling.connector.crmserviceclient

File: cds/orgsvc/C#/DiscoveryService/SampleMethods.cs [hunk 2]
```diff
@@ -14,39 +14,39 @@ namespace PowerApps.Samples
     public partial class SampleProgram
     {
         /// <summary>
-        /// Gets organization data for user in all regions when data center is unknown
+        /// Gets organization data for user in commercial region cloud is unknown
         /// </summary>
```
Comment (phecke) [inline cds/orgsvc/C#/DiscoveryService/SampleMethods.cs:18]: Not quite a sentence. Recommend simply "Gets organization data for user."

Comment (MattB-msft) [reply cds/orgsvc/C#/DiscoveryService/SampleMethods.cs:18]: Fixed

File: cds/orgsvc/C#/DiscoveryService/README.md (hunk 1 above)
Comment (MattB-msft) [reply cds/orgsvc/C#/DiscoveryService/README.md:15]: Fixed

**Pull request merged**